roles/pulp-webserver/defaults/main.yml

```diff
@@ -5,3 +5,5 @@ pulp_content_port: 24816
 pulp_api_host: 127.0.0.1
 pulp_api_port: 24817
 pulp_configure_firewall: auto
+pulp_static_url: 'assets'
+pulp_webserver_static_dir: "/var/lib/pulp/{{ pulp_static_url | regex_replace('^\\/|\\/$', '') }}/"
```

Comment:
Should this be relative to `pulp_user_home`?

`pulp_cache_dir` is not, it defaults to `/var/lib/pulp/tmp`.

Reply:
good question! This time the only thing I took as reference was `pulpcore` :
https://github.com/pulp/pulpcore/blob/master/pulpcore/app/settings.py#L31-L37
I'm not sure how `MEDIA_ROOT` is related to `pulp_user_home`
@dkliban @bmbouter could you help with this question?

Reply:
MEDIA_ROOT should be the same as pulp_user_home.

https://github.com/pulp/ansible-pulp/blob/master/roles/pulp/defaults/main.yml#L32